### ISSUE TITLE
File system issues (read-only file system)

### ISSUE THREAD

**Issue by chrisgorgo** [OpenNeuroOrg/openneuro#122]
Experienced on prod, only for a subset of jobs.
```
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/utils.py", line 1076, in make_output_dir
    os.makedirs(outdir)
  File "/usr/local/miniconda/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
OSError: [Errno 30] Read-only file system: '/usr/local/src/mriqc/work/workflow_enumerator/funcMRIQC/_in_file_..snapshot..data..sub-051..func..sub-051_task-training_run-02_bold.nii.gz/compute_tsnr'
```

```
Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/utils.py", line 1076, in make_output_dir
    os.makedirs(outdir)
  File "/usr/local/miniconda/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
OSError: [Errno 30] Read-only file system: '/usr/local/src/mriqc/work/workflow_enumerator/funcMRIQC/_in_file_..snapshot..data..sub-051..func..sub-051_task-training_run-02_bold.nii.gz/ICA'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/multiproc.py", line 52, in run_node
    result['result'] = node.run(updatehash=updatehash)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 365, in run
    outdir = make_output_dir(outdir)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/utils.py", line 1080, in make_output_dir
    raise OSError('Could not create %s', outdir)
OSError: [Errno Could not create %s] /usr/local/src/mriqc/work/workflow_enumerator/funcMRIQC/_in_file_..snapshot..data..sub-051..func..sub-051_task-training_run-02_bold.nii.gz/ICA
```

https://openneuro.org/datasets/ds001099/versions/00001?app=MRIQC&version=40&job=f44d912f-0308-4de1-bdc3-f59f4838d7b0

**Comment by chrisgorgo** [comment]
Run a similar job and got the same issue: https://openneuro.org/datasets/ds001099/versions/00001?app=MRIQC&version=42&job=e0600bfd-4988-42b5-8596-b8cf748d0b0e

**Comment by nellh** [comment]
The hosts had been shut down before I could look at the earlier instance of this but found these containers and the filesystem at the host level looks fine. Seeing if I can reproduce this outside of ECS.

**Comment by nellh** [comment]
I've updated bids-app-host to include some extra debugging info for this and to simplify when docker cleanup runs, since some hosts are still getting close to running out of disk and metadata limits might be causing this. I'm testing this on dev today and will update the stable tag once testing is finished.

**Comment by nellh** [comment]
This bids-app-host update (v0.7.11) is tagged, new jobs will start picking this up.

One more thing I'm looking into is upping the metadata space, since that is the limit we have hit here. Reducing the objects stored indirectly helps with that.

**Comment by chrisgorgo** [comment]
Rerunning - will let you know how it went.

**Comment by nellh** [comment]
@chrisfilo Sorry, it's actually updated now. v0.7.11 was updated but not latest-stable.

**Comment by nellh** [comment]
There is no metadata storage now that we're using Overlay 2 and if this was to reoccur, it would be a new issue.
